core/src/main/java/com/devonfw/tools/solicitor/reader/npmlicensecrawler/NpmLicenseCrawlerReader.java

```diff
@@ -69,12 +69,12 @@ public Set<String> getSupportedTypes() {
   public void readInventory(String type, String sourceUrl, Application application, UsagePattern usagePattern,
       String repoType, Map<String, String> configuration) {
 
-    this.deprecationChecker.check(true, "Use of Reader of type '" + SUPPORTED_TYPE
-        + "' is deprecated, use 'npm-license-checker' instead. See https://github.com/devonfw/solicitor/issues/125");
+    this.deprecationChecker.check(false, "Use of Reader of type '" + SUPPORTED_TYPE
+        + "' is deprecated, use 'npm-license-checker' instead. See https://github.com/devonfw/solicitor/issues/263");
```

Comment:
We should list both tickets as the first gives some background and the second indicates that it is now stage 2.